src/main/java/teammates/common/datatransfer/FeedbackTextQuestionDetails.java

```diff
@@ -83,6 +83,10 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
             PageData pageData,
             FeedbackSessionResultsBundle bundle,
             String view) {
+        if (responses.isEmpty()) {
+            return "";
+        }
+        
```

Comment:
This should probably be below the `@SuppressWarnings("unused")`.
On that note, maybe the warning suppression and TODO can be moved above the method signature instead.